src/winter_onboarding_2021/fleet_management_service/db/user.clj

```diff
@@ -1,6 +1,6 @@
 (ns winter-onboarding-2021.fleet-management-service.db.user
   (:require [honey.sql :as sql]
-            [honey.sql.helpers :refer [update set where]]
+            [honey.sql.helpers :as h]
```

Comment:
I feel abbreviated h is short, simple and since we are using `h` quiet number of times. If we are using an alias sql-helpers , it will make the function look more bulky. What do u think?